### PR TITLE
misc: Update and fix contribution guide

### PR DESCRIPTION
**Pull request by vincent-pochet**
Update the contribution guide to improve the contribution experience:
- Fix links to community slack
- Fix link to feature request tool
- Fix issues and pull request labels

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,32 +52,34 @@ Here's a list of Lago's elements:
...
 
-Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out the required [template](https://github.com/getlago/lago-python-client/issues/new?assignees=&labels=%F0%9F%90%9E+bug&template=bug.md&title=%5BBUG%5D%3A+), the information it asks for helps us resolve issues faster.
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out the required [template](https://github.com/getlago/lago-nodejs-client/issues/new?assignees=&labels=%F0%9F%90%9E+bug&template=bug_report.md&title=%5BBUG%5D%3A+), the information it asks for helps us resolve issues faster.
```
Comment (ansmonjol) [inline CONTRIBUTING.md:76]: Why does it switch to lago-nodejs-client ? Which is deprecated BTW

Comment (ansmonjol) [reply CONTRIBUTING.md:76]: Also concerns other occurrences bellow

Comment (vincent-pochet) [reply CONTRIBUTING.md:76]: Oh! Good catch!

**Review by rsempe — APPROVED** [review]
🙌

**Pull request merged**